### PR TITLE
[EPMPAOS-4946] Add iam v5 proto and remove SM NodeConfiguration

### PR DESCRIPTION
**Pull request by mlohvynenko**

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -1,9 +1,7 @@
-BasedOnStyle: WebKit
+BasedOnStyle: Google
```
Comment (mlohvynenko) [inline .clang-format:1]: I've made these changes cause origin version of the clang-format file doesn't format code as we have in existing proto files.

**Review by mykola-kobets-epam — APPROVED** [review]

File: proto/iamanager/v5/iamanager.proto [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    repeated NodeAttribute attrs      = 10;
+}
+
+message StartProvisioningRequest {
+    string node_id  = 1;
+    string password = 2;
+}
+
+message StartProvisioningResponse {
+    ErrorInfo eror_info = 1;
```
[the remark this answers is not in the document]
Comment (mlohvynenko) [reply proto/iamanager/v5/iamanager.proto]: Thanks, fixed

File: proto/iamanager/v5/iamanager.proto [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    string status                     = 4;
+    string os_type                    = 5;
+    repeated CPUInfo cpus             = 6;
+    float max_dmips                   = 7;
+    uint64 total_ram                  = 8;
+    repeated PartitionInfo partitions = 9;
+    repeated NodeAttribute attrs      = 10;
+}
+
+message StartProvisioningRequest {
```
Comment (al1img) [inline proto/iamanager/v5/iamanager.proto:200]: For all the messages put Response message just after corresponding request.

Comment (mlohvynenko) [reply proto/iamanager/v5/iamanager.proto:208]: Done

File: proto/iamanager/v5/iamanager.proto [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+    string functional_server_id = 2;
+}
+
+message InstanceIdent {
+    string service_id = 1;
+    string subject_id = 2;
+    uint64 instance   = 3;
+}
+
+message PermissionsResponse {
```
Comment (al1img) [inline proto/iamanager/v5/iamanager.proto:126]: Move after request

Comment (mlohvynenko) [reply proto/iamanager/v5/iamanager.proto]: done

**Pull request merged**